src/main/java/org/ecocean/CommonConfiguration.java

```diff
@@ -1021,4 +1021,25 @@ public static String getWildbookCommunityURL(String context) {
     return getProperty("wildbookCommunityUrl", context).trim();
   }
 
+  /**
+   * Retrieves the sessionWarningTime.
+   *
+   * @param context Webapp context
+   * @return The sessionWarningTime
+   */
+
+    public static int getSessionWarningTime(String context) {
+
+    int def = 20;
```

Comment:
can we leave a comment (for future reference) that this value is in minutes? maybe also in the properties file for good luck.

Reply:
thanks for suggestion i have updated the docstring and added comment in properties